src/problem-analysis/presentation.tex

```diff
@@ -0,0 +1,28 @@
+% textidote: ignore begin
+\section{Presentation of the problem}\label{sec:presentation-of-the-problem}
+% textidote: ignore end
+
+Several shortcomings from NOVA have been documented in the transcript below.
+The most noticeable is the issues regarding the~\acrshort{epos} insights and how they are not elaborate enough in its
+current state.
+
+% textidote: ignore begin
+\begin{displayquote}
+% textidote: ignore end
+    \textquote[\cite{nova}]{As NOVA is a relatively new business,
```

Comment:
Ok, but now every time we use block quotes, we have to enqote the quote manually. Why not use the blockquote hook from the documentation that I posted during my last review?

Reply:
@MadsHeilmann Then just wrap the quote using `\blockcquote`:
```suggestion
    \blockcquote{nova}{As NOVA is a relatively new business,
```

Reply:
Also, remove the `displayquote` environment.

Reply:
That's why you should do as I wrote previously and remove the `displayquote` environment and use `\blockcquote`

Reply:
But as i said that there are no """" when only using `\blockcquote`

Reply:
As I see it the easiest way is maybe just to use `\blockcquote{\textquote{}}`

Reply:
Am I the only one who thinks that there needs to be quotation marks around the quote? It is then both clearer that it is a quote and also prettier in my opinion.

Reply:
aggressive beskeder over github gavner ikke rigtigt nogle. Det er for den sags skyld også en gruppe opgave, så der er ikke en som bestemmer alene hvordan ting skal gøres.

Reply:
@SebastianMygind
1. Der er ikke noget aggressivt i mine kommentarer, men sarkasme, undtagen "End of discussion.", da det er nødvendigt at sætte grænser når der ikke er mere man kan tilføje.
2. Jeg har aldrig afvist beskeder på Teams. Tværtimod svarer jeg hurtig på Teams, skulle der være spørgsmål.
3. Jeg har i alt givet løsningsforslag og konkrete løsninger 7 gange i denne PR, endda med `Suggested change` funktionen og links til dokumentation, og kun efter 14 beskeder blev de implementeret. Se punkt 1.
4. Problemet her handlede ikke om grupperelaterede spørgsmål, såsom indhold og eksekvering af rapporten og projektet, men om brugen af LaTeX.

Reply:
I have now made it a blockquote. We can always chance it later if necessary